module/purpose/Goalie/src/Goalie.cpp

```diff
@@ -0,0 +1,102 @@
+#include "Goalie.hpp"
+
+#include "extension/Behaviour.hpp"
+#include "extension/Configuration.hpp"
+
+#include "message/input/GameState.hpp"
+#include "message/planning/LookAround.hpp"
+#include "message/purpose/Goalie.hpp"
+#include "message/strategy/DiveToBall.hpp"
+#include "message/strategy/LookAtFeature.hpp"
+#include "message/strategy/Ready.hpp"
```

Comment:
can be removed, this was the old version of ready before we had walk to field position